www/images/icon-webvitals-testing.svg

```diff
@@ -1,6 +1 @@
-<svg viewBox="0 0 42 22" xmlns="http://www.w3.org/2000/svg">
-<g stroke="#FFF" fill="none" fill-rule="evenodd">
-<rect x="1" y="1" width="40" height="20" rx="4" style="fill:rgb(211,233,255);stroke:rgb(211,233,255)"/>
-<text x="5.5" y="15" letter-spacing="1.1" style="font-family:sans-serif;font-size:12px;stroke:rgb(18,29,53);fill:rgb(18,29,53)">NEW</text>
-</g>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 42 22"><rect width="41" height="21" x=".5" y=".5" fill="#d3e9ff" rx="4.5"/><text x="5.5" y="15" fill="#121d35" font-weight="700" font-family="sans-serif" font-size="12" textLength="32">NEW</text></svg>
```

Comment:
Rich diff on these two are pretty far off, would want design clearance

Reply:
Hm, I could have sworn it looked more similar. I’ll check if this is true in-browser or something with GitHub’s rich diffs

Reply:
GitHub is embedding them as regular old `<img src=….svg>`, so can’t be that. Looks like I assumed exchanging the `stroke` for an increase to the `font-weight` was the same, but that was wrong. Will fix

Reply:
Should look the same now